### DISCUSSION TITLE
What events will CASA be a part of in 2023?

### DISCUSSION THREAD

**Discussion by bmann** [ChainAgnostic/CASA#59] [General]
Here is a list of Ethereum events roughly planned for 2023 https://docs.google.com/spreadsheets/d/1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ/edit#gid=1284373176

What other blockchain ecosystems might CASA be present at?

- Open Wallet Foundation
- IPFS Thing / IPFS Camp
- DWeb
- DIF
- identity related
- Browser related

Do we have any concrete plans yet for 2023 events schedule and CASA attendance?

**Comment by ligi** [comment]
Might do something small at cccamp as there might be a DWeb village.

**Reply by bumblefudge** [reply]
Yes!  I am very supportive of [this idea](https://leanderwattig.com/event/chaos-communication-camp-2023/) and hope to still be in Berlin that late in the summer!

**Comment by ritave** [comment]
A random idea, since I've never been to, but maybe there's some common ground with CASA and [CCC](https://en.wikipedia.org/wiki/Chaos_Communication_Congress).
